app.py

```
@@ -329,6 +329,50 @@ def dmx_worker():
+        # Stop the current worker
+        running = False
+        if dmx_receiver:
+            dmx_receiver.stop()
+            dmx_receiver.reset_stats()
+        
+        # Wait for thread to finish
+        if dmx_thread and dmx_thread.is_alive():
+            dmx_thread.join(timeout=1.0)
+        
+        # Restart with updated mappings
+        sacn_bind_ip = None if _normalize_interface_ip(sacn_interface) == '0.0.0.0' else sacn_interface
```

Ruff (0.14.10)

Possible binding to all interfaces

(S104)